ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73.yaml

```diff
@@ -288,6 +292,16 @@ tests:
     test:
     - ref: stackrox-e2e
   timeout: 5h0m0s
+- as: gke-race-condition-qa-e2e-tests
+  steps:
+    test:
+    - ref: stackrox-e2e
+  timeout: 5h0m0s
+- as: gke-version-compatibility-tests
```

Comment:
They were partial at 3.73 cut-off time so we could try... Although I did not enable in my https://github.com/openshift/release/pull/36303